src/displayapp/screens/settings/QuickSettings.cpp

```diff
@@ -9,13 +9,17 @@ using namespace Pinetime::Applications::Screens;
 namespace {
   void ButtonEventHandler(lv_obj_t* obj, lv_event_t event) {
     auto* screen = static_cast<QuickSettings*>(obj->user_data);
-    screen->OnButtonEvent(obj, event);
+    if (event == LV_EVENT_CLICKED) {
+      screen->OnButtonEvent(obj);
+    }
   }
 
   void lv_update_task(struct _lv_task_t* task) {
     auto* user_data = static_cast<QuickSettings*>(task->user_data);
     user_data->UpdateScreen();
   }
+
+  constexpr uint8_t specialState = 0x40;
```

Comment:
The name of this constant does not provide enough clue about its meaning. If I understand correctly, it's  used to define a 3rd state to the button (default, checked, "special").

To be more expressive, we can maybe try this : 

```
static constexpr uint8_t notificationOnState = LV_STATE_CHECKED;
static constexpr uint8_t notificationOffState = LV_STATE_DEFAULT;
static constexpr uint8_t notificationSleepState = 0x40; // define an additional state to the button to handle Sleep mode
```

And why not use a `enum class` ?
```
enum class ButtonState : uint8_t {
  On = LV_STATE_CHECKED,
  Off = LV_STATE_DEFAULT,
  Sleep = 0x40; // define an additional state to the button to handle Sleep mode
}
```